worlds/wargroove/docs/wargroove_en.md

```diff
@@ -0,0 +1,81 @@
+
+1. Shutdown Wargroove if it is open.
+2. Start the ArchipelagoWargrooveClient.exe from the archipelago installation. 
+This should install the mod and campaign for you.
+3. Start Wargroove.
+
+## Verify the campaign can be loaded
+
+1. Start Wargroove from Steam
+2. Go to `Story->Campaign->Custom->Archipelago` and click play. You should see the first level
+
+## Starting a Multiworld game
+
+1. Start the Wargroove Client and connect to the server. Enter your username from your 
+[settings file](/games/Wargroove/player-settings)
+2. Start Wargroove and play the Archipelago campaign by going to `Story->Campaign->Custom->Archipelago`
+
+## Ending a Multiworld game
+It is strongly recommended that after finishing a multiworld game to delete campaign progress.
+This can be done by going to the level selection screen in the Archipelago campaign, hitting `ESC` and clicking the 
+`Delete Progress` button. The main menu should now be visible.
+
+## Updating to a new version of the Wargroove mod or downloading new campaign files
+Please delete your campaign progress by going to the level selection screen in the Archipelago campaign, 
+hitting `ESC` and clicking the `Delete Progress` button.
+
+Follow the `Installing the Archipelago Wargroove Mod and Campaign files` steps again, but look for the latest version
+ to download. In addition, follow the steps outlined in `Wargroove crashes when trying to run the Archipelago campaign`
+when attempting to update the campaign files and the mod.
+
+## Troubleshooting
+
+### The game is too hard
+`Go to the campaign overview screen->Hit escape on the keyboard->Click adjust difficulty->Adjust the setttings`
+
+### The mod doesn't load
+Double-check the mod installation under `%appdata%\Chucklefish\Wargroove\mods` there should be 3 `.dat` files in 
+`%appdata%\Chucklefish\Wargroove\mods\ArchipelagoMod`. Otherwise, follow 
+`Installing the Archipelago Wargroove Mod and Campaign files` steps once more.
+
+### Wargroove crashes or there is a lua error
+Wargroove is finicky, but there could be several causes for this. If it happens often or can be reproduced, 
+please submit a bug report.
+
+### Wargroove crashes when trying to run the Archipelago campaign
+This is caused by not deleting campaign progress before updating the mod and campaign files.
+1. Go to `Custom Content->Create->Campaign->Archipelago->Edit` and attempt to update the mod.
+2. Wargroove will give an error message.
+3. Go back to `Custom Content->Create->Campaign->Archipelago->Edit` and attempt to update the mod again.
+4. Wargroove crashes.
+5. Go back to `Custom Content->Create->Campaign->Archipelago->Edit` and attempt to update the mod again.
+6. In the edit menu, hit `ESC` and click `Delete Progress`.
+7. If the above steps do not allow you to start the campaign from `Story->Campaign->Custom->Archipelago` replace 
+`playerProgress` and `playerProgress.bak` with your previously backed up files.
```

Comment:
This is because the campaign save data exists in a playerProgress file which contains the save data for all of the user's campaigns. This includes the vanilla and double trouble campaigns. The steps above allow the user to utilize an updated campaign or mod without receiving an error when trying to start the campaign again. Unfortunately, I haven't found any other way to fix the error when updating the campaign.

Reply:
Okay, fair. I suggest adding instructions earlier in the guide about backing up save files, since you seem to expect that the player _has_ such backups.

Reply:
Ah I must've had those instructions in there earlier, but removed them at one point. I added a new section called `Backup playerProgress files`